client/packages/requisitions/src/RnRForms/DetailView/RnRFormLine.tsx

```diff
@@ -310,57 +310,73 @@ export const RnRFormLine = ({
 const RnRNumberCell = ({
   value,
   disabled,
-  readOnly,
   onChange,
   textColor,
+  backgroundColor: inputBackgroundColor,
   max,
   error,
   allowNegative,
 }: {
   value: number;
   error?: boolean;
   disabled?: boolean;
-  readOnly?: boolean;
-  onChange: (val: number) => void;
+  onChange?: (val: number) => void;
   textColor?: string;
+  backgroundColor?: string;
   max?: number;
   allowNegative?: boolean;
 }) => {
-  const theme = useTheme();
-  const backgroundColor = readOnly ? theme.palette.background.drawer : 'white';
-
   const [buffer, setBuffer] = useBufferState<number | undefined>(
     NumUtils.round(value)
   );
 
+  const backgroundColor = inputBackgroundColor ?? 'white';
+
   return (
     <td style={{ backgroundColor }}>
       <Tooltip title={value === buffer ? '' : value}>
-        <NumericTextInput
-          InputProps={{
-            sx: {
-              backgroundColor,
-              '& .MuiInput-input': {
-                WebkitTextFillColor: textColor,
+        {disabled || !onChange ? (
+          <p
+            style={{
+              padding: '8px',
+              textAlign: 'right',
+              color: textColor,
+            }}
+          >
+            {buffer}
+          </p>
```

Comment:
Just load a <p> if not editable anyway

Reply:
I added error state to this further up downstream